src/pages/Mentors/index.tsx

```diff
@@ -59,69 +143,149 @@ const Mentors = () => {
+                    checked={selectedCountries.length === 0}
+                    onChange={() => setSelectedCountries([])}
+                    className="form-checkbox h-4 w-4 text-blue-500"
+                  />
+                  <span className="text-gray-700">All</span>
+                </label>
+                {uniqueCountries.map((country) => (
+                  <label key={country} className="flex items-center space-x-2">
+                    <input
+                      type="checkbox"
+                      checked={selectedCountries.includes(country)}
+                      onChange={() => handleCountryChange(country)}
+                      className="form-checkbox h-4 w-4 text-blue-500"
+                    />
+                    <span className="text-gray-700">{country}</span>
+                  </label>
+                ))}
+              </div>
+            )}
           </div>
-        </div>
 
-        <div className="mb-4 w-full flex justify-end">
-          <button
-            onClick={handleSortAZ}
-            className="bg-blue-500 text-white px-2 py-1 rounded border border-blue-500 text-xs mr-6"
-          >
-            Sort A-Z
-          </button>
-        </div>
+          <div className="mb-6">
+            <p className="font-medium mb-2 text-gray-700 flex items-center">
+              <button
+                className=" text-gray-700 hover:text-blue-500"
+                onClick={toggleSlotsDropdown}
+              >
+                Available Slots {isSlotsDropdownOpen ? '-' : ' +'}
+              </button>
+            </p>
 
-        {sortedMentors.length > 0 ? (
-          <div className="flex-grow grid grid-cols-1 sm:grid-cols-2 md:grid-cols-3 lg:grid-cols-5 xl:grid-cols-6 gap-8 items-start">
-            {sortedMentors.map((mentor) => (
-              <MentorCard key={mentor.uuid} mentor={mentor} />
-            ))}
+            {isSlotsDropdownOpen && (
+              <div className="flex flex-col gap-3 text-sm">
+                <label className="flex items-center space-x-2">
+                  <input
+                    type="checkbox"
+                    checked={selectedAvailableSlots.length === 0}
+                    onChange={() => setSelectedAvailableSlots([])}
+                    className="form-checkbox h-4 w-4 text-blue-500"
+                  />
+                  <span className="text-gray-700">All</span>
+                </label>
+                {uniqueAvailableSlots.map((slots) => (
+                  <label key={slots} className="flex items-center space-x-2">
+                    <input
+                      type="checkbox"
+                      checked={selectedAvailableSlots.includes(slots)}
+                      onChange={() => handleAvailableSlotsChange(slots)}
+                      className="form-checkbox h-4 w-4 text-blue-500"
+                    />
+                    <span className="text-gray-700">
+                      {slots} Slots Available
```

Comment:
Slots should be a slider like this,

<img width="319" alt="image" src="https://github.com/user-attachments/assets/8d88aaf0-abab-499e-b8a2-8b8ac051021f">